app_dart/integration_test/validate_all_ci_configs_test.dart

```diff
@@ -0,0 +1,65 @@
+// Copyright 2021 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io' as io;
+
+import 'package:file/file.dart';
+import 'package:file/local.dart';
+import 'package:meta/meta.dart';
+import 'package:test/test.dart';
+import 'package:yaml/yaml.dart';
+
+import 'package:cocoon_service/cocoon_service.dart';
+
+class Repo {
+  const Repo({
+    @required this.name,
+    @required this.remoteUrl,
+    this.ref = 'master',
```

Comment:
Future work: How should we collect the branches that we also validate on?

Reply:
these can be implemented as separate entries in the `allRepos` list.